server/messaging/reporting.py

```
@@ -226,11 +226,12 @@ def get_section_condition(section):
 
         try:
             json_obj = db.get_table_as_json(sqlQuery, parameters)
+            data = apply_timezone_to_json(json_obj, section)
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)